torch/autograd/functional.py

```diff
@@ -545,7 +595,7 @@ def jac_func(*inp):
         _check_requires_grad(jac, "jacobian", strict=strict)
         return jac
 
-    res = jacobian(jac_func, inputs, create_graph=create_graph, strict=strict)
+    res = jacobian(jac_func, inputs, create_graph=create_graph, strict=strict, vectorize=vectorize)
```

Comment:
Should we also `vectorize` the call to jacobian in `jac_func` ?
Or that is not supported by vmap?

Reply:
The function that we are computing a hessian over only has 1 output with 1 element. Therefore the `jacobian` call in `jac_func` is equivalent to a `vjp` call and doesn't benefit from `vectorize=True`.

Reply:
Ho right, I forgot that we have this constraint! Sounds good!